spec/unit/driverProviders/direct_test.js

```diff
@@ -0,0 +1,69 @@
+
+var fs = require('fs'),
+    os = require('os'),
+    path = require('path');
+var BrowserError = require('../../../built/exitCodes').BrowserError,
+    Logger = require('../../../built/logger2').Logger,
+    WriteTo = require('../../../built/logger2').WriteTo;
+
+describe('direct connect', function() {
+  beforeEach(function() {
+    Logger.setWrite(WriteTo.NONE);
+  });
+
+  afterEach(function() {
+    Logger.setWrite(WriteTo.CONSOLE);
+  });
+
+  describe('without the selenium driver', function() {
+    it('should throw an error if no drivers are present', function() {
+      var config = {
+        directConnect: true,
+        capabilities: { browserName: 'chrome' },
+        chromeDriver: '/foo/bar/chromedriver'
+      };
+      var errorFound = false;
+      try {
+        webdriver = require('../../../built/driverProviders/direct')(config);
+        webdriver.getNewDriver();
+      } catch(e) {
+        errorFound = true;
+        expect(e.code).toBe(BrowserError.CODE);
+      }
+      expect(errorFound).toBe(true);
+    });
+  });
+
+  describe('with selenium drivers', function() {
+    var chromedriver = '';
+    beforeEach(function() {
+      // add files to selenium folder
```

Comment:
Why is it important that the file have the correct version? Aren't we just testing that if the file exists but isn't a proper driver, we get the right error code?

I think this should either just use a bogus file that we create (like os.tpdir() + '/foo/bar') or not specify `chromeDriver` in the config and let the driverProvider use its normal logic.